content.md

```diff
@@ -1,125 +1,126 @@
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools 
+  across EdgeCast to create consistent processes and tooling in line with best practices. 
+  - **Evangelized SRE best practices to leadership.** Educated senior directors and VPs on the value of **SLIs/SLOs, error budgets, 
+    consistent tooling and practices**, etc. to build a better data-driven platform. 
+  - **Established SLIs and SLOs across teams.** Having established the value of SL\[IO\]s, I took 
+    on the large task of meeting with service teams across the org to understand their services and dependencies. I explained what makes useful, actionable SLIs, and coached team representatives through writing and instrumenting new SLIs and SLOs in line with these standards. 
+    This enabled the platform as a whole to move away from a reactive, incident-driven maintenance model towards proactive, error-budget-driven development. 
+  - **Created and documented escalation best practices for oncall engineers.** 
+    - **Establiblished standard oncall response SLAs and expectations.** After noticing differing and undocumented oncall response across teams different teams, I took on the task of creating common expectations across teams. I empaneled meetings with various stakeholders (NOC, service owners, leadership) to understand each team's implicit expectations for escalations (both their own and other teams'). Based on this input, I set up example service tiers with **specific SLAs around response time**, **definitions** for what "response" entailed, and **standards** for how to publish each team's oncall SLA and escalation procedures where they could be seen by others. 
+    - When onboarding the main CDN dev team onto the a shared rotation with SREs, I documented oncall expectations in line with the standards I created above. 
+- **Provided technical solutions to reliability problems.** 
+  - Moved configuration integration testing into its own service, independent from code testing, resulting in a more stable and consistent build/certification pipeline. 
+  - **Designed solution for a major reliability problem** in which a single node failures caused delays and buffering across all videos and large files delivered. Worked with the CDN Devs to implement this solution to identify and route around failed nodes even before they were removed from the working pool. 
+ 
+ 
+### Google, Inc. - **Site Reliability Engineer** - *February 2008 - February 2020* 
+ 
+During my tenure as an SRE, I've built and run Google enterprise infrastructure using open source tools on vanilla-ish Linux, 
```

Comment:
During my tenure as an SRE, I built and ran Google enterprise infrastructure on a range of platforms including using open source tools on Linux, Google’s production infrastructure, and Google Cloud Platform. Provided oncall support with 5- and 30-minute response SLAs, and continously evaluated issues and responses and built tooling and processes to streamline operations and automate away toil